draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -368,6 +368,11 @@ cookie with a path of /c,d=e.
 User agents MAY ignore Set-Cookie header fields based on response status codes or
 the user agent's cookie policy (see {{ignoring-cookies}}).
 
+Note: A cookie's octets MUST be processed as {{USASCII}} characters. While
+it's possible a non-HTTP API could pass a set-cookie-string with one or more
+non-{{USASCII}} characters, no attempt should be made to interpret these octets
+as anything other than {{USASCII}} characters.
```

Comment:
This is meant to address

>RFC 9110 prescribes that HTTP headers be in US-ASCII.  However the
existence of non-HTTP APIs for the cookie store raises the possibility
that a larger set of characters might be supported.  As far as I can
tell, this document does not specify the universe of characters.

I'm trying to communicate that when we do "look at" the octets that we process them as ascii, and to basically avoid trying to interpret anything like
`document.cookie = "foo=🍪"

Reply:
I think this is consistent with the RFC we're updating, and that it's perfectly reasonable to leave clearer integration with HTML and DOM APIs to the `tre` layering document in progress.

Reply:
Ok. Just to be clear, this text LGTY?